joomla/cassiopeia
# Replace template_thumbnail for cassiopeia

Pull Request for Issue #174 .

### Summary of Changes
A new screen of the new cassiopeia layout

### Testing Instructions
Apply the patch and see the template in the backend
![grafik](https://user-images.githubusercontent.com/1035262/98485760-0e0a1280-2219-11eb-8403-dda94e964188.png)

with Preview
![grafik](https://user-images.githubusercontent.com/1035262/99190682-4bfebd80-2768-11eb-924c-9818acb511a1.png)


### Expected result

### Actual result
Tumbnail of the first version of cassiopeia


### Documentation Changes Required 
yes, screenshots.

### Note 
Designer's Help would be highly appreciated
